climada/hazard/trop_cyclone.py

```diff
@@ -1373,7 +1397,7 @@ def _B_holland_1980(si_track: xr.Dataset):  # pylint: disable=invalid-name
 def _x_holland_2010(
     si_track: xr.Dataset,
     d_centr: np.ndarray,
-    close_centr: np.ndarray,
+    mask_centr_close: np.ndarray,
```

Comment:
The variable name `close_centr` is used in many places in `trop_cyclone.py`, and I found it confusing because the name sounds like the data is describing centroids (i.e. coordinates of points), but instead, it's just a mask (a bool array) over the set of centroids. The renaming should clarify this.